html/changelogs/sidveld-augment-translator.yml

```diff
@@ -0,0 +1,6 @@
+author: SidVeld
+
+delete-after: True
+
+changes:
+  - rscadd: "Added an augment translator that can translate human languages."
```

Comment:
i think be a little more explicit here so people don't get confused and angry. maybe something like `Added an universal translator augment that lets the user understand but not speak human languages.`